metricbeat/helper/prometheus/metric.go

```diff
@@ -135,28 +135,33 @@ func (m *commonMetric) GetField() string {
 func (m *commonMetric) GetValue(metric *dto.Metric) interface{} {
```

Comment:
We could do some refactors to split this method in smaller methods and to reuse the check code, but nothing to do in this PR.